types/generated/contentTypes.d.ts

```diff
@@ -640,8 +640,13 @@ export interface PluginUsersPermissionsUser extends Schema.CollectionType {
       'oneToOne',
       'api::profile.profile'
     >;
-    discordId: Attribute.String & Attribute.Private;
-    googleId: Attribute.String & Attribute.Private;
+    time_capsules: Attribute.Relation<
```

Comment:
We have removed Time Capsule in https://github.com/dev-launchers/strapiv4/pull/146/files#diff-7d06fadd606542cc6ab66af65e7560f74b247e444f5abb8f347063f53a2ac30aL643